static/app/views/alerts/rules/metric/details/body.tsx

```diff
@@ -172,6 +177,22 @@ export default class DetailsBody extends Component<Props> {
           )}
         <Layout.Body>
           <Layout.Main>
+            {isSnoozed && (
+              <Alert showIcon>
+                {ruleActionCategory === RuleActionsCategories.NO_DEFAULT
+                  ? tct(
+                      "[creator] muted this alert so these notifications won't be sent in the future.",
+                      {creator: rule.snoozeCreatedBy}
+                    )
+                  : tct(
+                      "[creator] muted this alert[forEveryone]so you won't get these notifications in the future.",
+                      {
+                        creator: rule.snoozeCreatedBy,
+                        forEveryone: rule.snoozeForEveryone ? ' for everyone ' : ' ',
+                      }
+                    )}
```

Comment:
dumb question but wouldn't metric alerts always be for everyone or are we looking for specific email my team -> this user is muted

Reply:
<img width="876" alt="Screenshot 2023-06-15 at 12 31 25 PM" src="https://github.com/getsentry/sentry/assets/29959063/92a1410d-4c3a-4ae3-8016-0c885a1ea490">

you can set it to email a user or yeah if it's a team it emails each user in the team and one person can mute it for themselves.